scheduler-server/src/test/java/gov/nasa/jpl/aerie/scheduler/server/services/SchedulingDSLCompilationServiceTests.java

```diff
@@ -31,8 +31,8 @@ class SchedulingDSLCompilationServiceTests {
   private static final PlanId PLAN_ID = new PlanId(1L);
   SchedulingDSLCompilationService schedulingDSLCompilationService;
 
-  @BeforeAll
-  void setUp() throws SchedulingDSLCompilationService.SchedulingDSLCompilationException, IOException {
+  @BeforeEach
```

Comment:
I was seeing issues in testing where multiple tests running in parallel were hitting the same executor node instance - which we have explicitly not supported at this point so they would hang waiting for a response that would never come. So this was to ensure that each had their own instance for the test to avoid collisions. I'm open to other solutions, but thought parallelizing the node process out of scope for this PR.

Reply:
I see - if that's the case I think your approach is fine. A little bit of searching on the web seems to suggest that JUnit doesn't run tests in parallel by default though, so I'd like to understand:
1. are our tests being run in parallel?
2. if not, can we understand the behavior you observed in your testing?

I have a hunch that there might be an actual bug that we're sidestepping by re-starting the subprocess between tests. I'll try to reproduce on my end as well

Reply:
See comment here: https://github.com/NASA-AMMOS/aerie/pull/134/files#r848808391

Reply:
Bad assumption on my part that most test runners run in parallel by default

Reply:
Addressed this with the break up of the commits. Thanks @mattdailis